webapp/components/wpt-metadata.js

```diff
@@ -266,7 +266,7 @@ class WPTMetadata extends PathInfo(LoadingState(PolymerElement)) {
       if (this.pathIsASubfolder) {
         curPath = curPath + '/';
       }
-      return curPath.startsWith(testname.substring(0, testname.length - 1));
+      return curPath.startsWith(testname.substring(0, testname.length - 1)) || this.isParentDir(curPath, testname.substring(0, testname.length - 2));
```

Comment:
IIUC, this change now shows wildcard metadata in the immediate parent. E.g., if you navigate `foo/`, you'll now see `foo/bar/*` at the bottom.

I like the new behaviour but the implementation seems a bit brittle regarding the trailing slash. Can we make `isParentDir` work both with and without the trailing slash, so that we can use `testname.substring(0, testname.length - 1)` (preferably through a new temporary variable) in both places?

Reply:
yea I had a feeling that my handling of trailing slash is brittle to begin with. But for your comment, it actually works with and without the trailing slash. See [test cases](https://github.com/web-platform-tests/wpt.fyi/pull/2046/files#diff-46af02d1b3b0dda7bc52ab60e2815e9eR88). I will use `testname.substring(0, testname.length - 1)` for consistency